packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -487,6 +486,11 @@ export function augmentStackTraceWithInitialElement(stackInfo: StackTrace, url?:
   return false
 }
 
+let reportFunction: unknown
+export function setReportFunction(newReportFunction: unknown) {
+  reportFunction = newReportFunction
+}
+
```

Comment:
This is quite ugly, but I have no better solution that comply to the "sideeffect" rule.